tox.ini

```diff
@@ -109,6 +109,7 @@ envlist =
     tornado_contrib-py{27,35,36,37,38,39}-tornado{44,45}
     tornado_contrib-py{37,38,39}-tornado{50,51,60,}
     tornado_contrib-py27-tornado{44,45}-futures{30,31,32,}
+    urllib3_contrib-{py27,py35,py36,py37}-urllib3{122,123,124,125}
```

Comment:
we should add to `riotfile.py` instead, we will eventually move everything off of `tox`.